packages/server/src/unstable-core-do-not-import/middleware.ts

```diff
@@ -203,7 +262,7 @@ export function createInputMiddleware<TInput>(parse: ParseFn<TInput>) {
             }
           : parsedInput;
 
-      return opts.next({ input: combinedInput });
+      return opts.next({ [nextInputSymbol]: combinedInput });
```

Comment:
So that was my question as I thought createInputMiddleware used this.

I'm not entirely convinced there's a solid reason to take away input overriding from userland, though I'm also not sure if it's used by anybody for anything useful